Assets/LeapMotion/Core/Scripts/Utils/BitConverterNonAlloc.cs

```diff
@@ -9,7 +9,7 @@
 
 using System;
 
-namespace Leap.Unity {
+namespace Leap.Unity.Tests {
```

Comment:
Do we want the BitConverter itself in the Tests namespace?

Reply:
No we do not!  Fixed!